### DISCUSSION TITLE
restarting nginx service without rebooting the lxc container itself

### DISCUSSION THREAD

**Discussion by Mrdindon** [tteck/Proxmox#1140] [Q&A] [answered]
totally beginner here,
I'm messing around with nginx (trying to get certificates pushed from my pfsense router to my nginx lxc using acme commands / scp) and I'm trying to understand how to restart nginx service without rebooting the lxc container itself. I tried with systemctl and service but that doesn't work. Any idea ?

**Comment by tteck** [comment, marked as answer]
`systemctl restart npm`
If you're using the Nginx Proxy Manager LXC from this repo
